index.html

```diff
@@ -43,6 +44,9 @@ <h1 class="nameTemp has-text-warning is-size-1 is-size-4-mobile"></h1>
                   <div id="history">
                     <br>
                     <h2 class="has-text-warning is-size-3 ">Previous Searches</h2>
+                    <strong class="shiftRight">
+                      <h4><em><b>CLEAR</b></em></h4>
+                    </strong>
```

Comment:
Can we update this to a link perhaps, so we get the hover over cursor? Also, let's add an id to that 'link' - or button if youd rather go with a button, but I think a link here works. Something like this...
```
<a class="shiftRight" id="clearLink"><em>CLEAR</em></a>
```
This way you can remove the <strong> tag as well and just place this under the Previous Searches h2.

Reply:
I would need assistance with the hover as i never used that code b4. Maybe i'll pass this to Mariam.

Reply:
Let me know if you want to walk through it together - we can pair program it out.